packages/core/addon/serializers/dashboard-widget.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+//@ts-ignore
+import AssetSerializer from './asset';
+import Model from '@ember-data/model';
+import { normalizeTableV2 } from './table';
+
+export default class DashboardWidgetSerializer extends AssetSerializer {
```

Comment:
Could this just extend the ReportSerializer since it looks like they cover the same functionality?

Reply:
They cover a similar functionality, but since widgets have `requests` it's slightly different and I didn't think it was enough code to justify reusing